app/boards/intel_adsp_ace30_ptl.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_COMP_ARIA=y
 CONFIG_COMP_CHAIN_DMA=y
 CONFIG_COMP_DRC=y
 CONFIG_COMP_KPB=y
+CONFIG_COMP_TESTER=m
```

Comment:
@lyakh Hmm, how does this now work. This will go to sof-ptl-openmodules.ri and loaded in production systems, right? Is this ok, or should we have some blacklist method to filter out modules like this from sof-ptl-openmodules.ri, or do we already have one?

Reply:
@kv2019i no, it won't go into openmodules, because the CMakeLists.txt file for this module has no `LIB openmodules` record, so it will be built as a separate module, and it's up to the packager whether or not to ship it with the release.

Reply:
@lyakh @kv2019i the module is not to be included in the release, **we wont sign the binary**. It is for a CI/regression testing that require additional internal code. 

Example: I want to use it soon for DP modules EDF scheduling test - by creating couple of test modules as DP and forcing each of them to have different deadlines and consume a precise number of CPU cycles. Than the modules can internally verify if they are called/preempted in the order they should be.
Such test maybe is not to be included in CI, but is should go to the regression test set to verify that EDF scheduling and deadline calculation.  Validation than can easily load the debug module and repeat the test automatically, using a standard build.

Reply:
Ack @lyakh @marcinszkudlinski this is understood and seems covered.